RELEASING.md

```diff
@@ -0,0 +1,6 @@
+# Snapshot
```

Comment:
OK I updated the file. All it takes to release a build is to run one of those gradle tasks, and within a minute the artifacts are available for everyone. You don't have to log in to some archaic system, and wait a few hours for artifacts to sync.